metadata/identifier/service_metadata_identifier.go

```diff
@@ -27,12 +27,12 @@ type ServiceMetadataIdentifier struct {
 	BaseMetadataIdentifier
 }
 
-// getIdentifierKey...
+// getIdentifierKey gets metadata identifier key by @mdi.protocol and @mdi.revision
 func (mdi *ServiceMetadataIdentifier) getIdentifierKey(params ...string) string {
-	return mdi.BaseMetadataIdentifier.getIdentifierKey(mdi.protocol + constant.KEY_REVISON_PREFIX + mdi.revision)
+	return mdi.BaseMetadataIdentifier.getIdentifierKey(mdi.protocol, constant.KEY_REVISON_PREFIX, mdi.revision)
```

Comment:
I think should not change `+` to `,` 

When
`
protocol="a"
revision="b"
`

In `BaseServiceMetadataIdentifier.getIdentifierKey` use `+` , result is `arevisionb`.

In `BaseServiceMetadataIdentifier.getIdentifierKey` use `,` , result is `a:revision:b`.